resdk/resolwe.py

```diff
@@ -37,9 +37,7 @@
 
 VERSION_NUMBER_BITS = (8, 10, 14)
 CHUNK_SIZE = 8000000  # 8MB
-DEFAULT_EMAIL = 'anonymous@genialis.com'
-DEFAULT_PASSWD = 'anonymous'
-DEFAULT_URL = 'https://dictyexpress.research.bcm.edu'
+DEFAULT_URL = 'http://localhost:8000'
```

Comment:
@mstajdohar which server should be here? Maybe QA? But it is experimental and we don't guarantee that it works.